### DISCUSSION TITLE
Commerce Sections Granular Permissions

### DISCUSSION THREAD

**Discussion by rjhewitt3** [craftcms/commerce#2400] [Ideas]
**What are you trying to do?**
Give view-only permissions to various sections of Commerce.

**What's your proposed solution?**
Add user permissions similar to what's available for Orders (ie view, edit, delete). There are many different layers that could be given here, like what we have for sections—but as a phase 1 of this, the limited options mentioned here would be sufficient. To clarify, I'm asking for the granular permissions on products, promotions, shipping, and taxes.

Worth noting here too, when a user has the "Manage orders" permission, but not the sub-permission "Edit orders" they can't access the individual order screen—they can only see the table view. Users with manage, but not edit, should be able to see the full order details. The table view access really doesn't give them much since there is limited space for columns, etc.

_Found a related, but different (more specific) issue: https://github.com/craftcms/commerce/issues/174_

**Comment by pdaleramirez** [comment]
[#174](https://github.com/craftcms/commerce/issues/174) - Related enhancement
